examples/menubar/menubar-2/menubar-2.html

```diff
@@ -136,8 +122,9 @@ <h2 id="ex1_label">Example</h2>
           Gettysburg Address</a>
       </p>
       <script type="text/javascript">
-          var menubar = new MenubarAction(document.getElementById('menubar1'));
-          menubar.init();
+          var menubar      = new MenubarAction(document.getElementById('menubar1'));
```

Comment:
I agree with @mcking65. JavaScript should be placed in a predictable, common location so that the logic of the system is evident. Sprinkling behavior into the code like this is certainly convenient, but it makes the examples more difficult to follow; it also makes it more difficult to refactor in the future.